modules/manage/partials/tiered-storage.adoc

```diff
@@ -843,6 +843,8 @@ For additional properties, see <<Tiered Storage configuration properties>>.
 * <<Enable Tiered Storage for a cluster>>
 * <<Enable Tiered Storage for specific topics>>
 
+When Tiered Storage is enabled on a topic already containing data, it will proceed to upload any existing data in that topic on local storage to the object store bucket. It will start from the earliest offset available on local disk.
```

Comment:
can this behavior be disabled?

Reply:
Good question. didn't sound like the behavior could be disabled, but I didn't specifically ask about that in the original slack thread. @andrwng can you weigh in here?

Reply:
This can't be disabled. Tiered storage today doesn't consider when in its lifespan TS has been enabled or disabled -- it only knows if it is enabled, it must upload everything it has that hasn't already been uploaded (based on understanding the highest uploaded so far, which in this case would be starting from 0).

FWIW toggling tiered storage on already active topics is not something we're generally comfortable advising. It leaves us open to weird behaviors, like gaps in our remote log. If we want to document this, can we caveat this by advising users to avoid toggling?

Reply:
@andrwng Does the phrasing you used work for this:
| When Tiered Storage is enabled on a topic already containing data, it will proceed to upload any existing data in that topic on local storage to the object store bucket. It will start from the earliest offset available on local disk. Enabling Tiered Storage on already active topics is not recommended as you may encounter gaps in the remote logs.
 
And would that plus @hcoyote 's suggestion in this [comment](https://github.com/redpanda-data/docs/pull/583#issuecomment-2195726797) be sufficient?
| To ensure all data is uploaded to Tiered Storage, you should enable remote write before any data is written to the topic. If you enable this after data has been written to the topic, only the data that currently exists on disk will be scheduled for uploading.

^ This would be added to the [Remote write](https://deploy-preview-583--redpanda-docs-preview.netlify.app/current/manage/tiered-storage/#remote-write) section.

Reply:
Let's add an explicit note:

> Toggling remote write on and off could lead to inconsistent data and data gaps in Tiered Storage for a topic. Redpanda discourages doing this.

Reply:
>Does the phrasing you used work for this:

SGTM!

>And would that plus @hcoyote 's suggestion in this https://github.com/redpanda-data/docs/pull/583#issuecomment-2195726797 be sufficient?

This is accurate. I'm not sure how detailed to get -- a natural follow-up question is "why wouldn't the data be on disk?" and to answer that admins would need to consider their local storage retention settings. Might be worth pointing to those too? OTOH given it's something we don't recommend anyway, not sure how best to convey that, if anything at all